Update Server-Setup-and-Maintenance-Linux.md

EpicTaru opened the pull request:
* Did my best (without having a Linux server to verify) to replace all Topaz connotations with the relevant LSB connotations.

* Verified/converted all markdown styles to Github Markdown.

Fixes https://github.com/LandSandBoat/server/issues/2473

EpicTaru committed:
Update Server-Setup-and-Maintenance-Linux.md

* Did my best (without having a Linus server to verify to replace all Topaz connotations with the relevant LSB connotations.

* Verified/converted all markdown styles to Github Markdown.

Closes https://github.com/LandSandBoat/server/issues/2473

WinterSolstice8 commented on Server-Setup-and-Maintenance-Linux.md:
```diff
@@ -118,50 +118,31 @@ cd ../sql
+Now we are going to enter the username and password for sql inside `network.lua`
 
-First, copy these config files to the conf folder:
+First, copy the .lua settings files from the `settings\default\` folder to the `settings\` folder:
```
backslash to forwardslashes (nix uses forward slash for directories)

WinterSolstice8 commented on Server-Setup-and-Maintenance-Linux.md:
````diff
@@ -118,50 +118,31 @@ cd ../sql
-```
-cp server/scripts/settings/default/* server/scripts/settings/
-```
+NOTE: Any .lua files in the `settings\default\` folder that you wish to update needs to first be **copied** from the `server\settings\default\` folder and put in the precedent folder (`server\settings\`). Any .lua file(s) not updated can remain in the `server\settings\default\` folder (the .lua files in the `server\settings\default` folder are used if updated ones are not found in the `server\set ...
````
backslash to fowardslash

WinterSolstice8 reviewed (changes requested):
Looks good other than the slashes

TeoTwawki commented on Server-Setup-and-Maintenance-Linux.md:
```diff
@@ -118,50 +118,31 @@ cd ../sql
-echo -e "\n#DB_VER: `git rev-parse --short=4 HEAD`" >> version.conf
-nano login.conf
+cd settings
+echo -e "\n#DB_VER: `git rev-parse --short=4 HEAD`" >> network.lua
```
version was moved into login settings

https://github.com/LandSandBoat/server/blob/base/settings/default/login.lua#L15

TeoTwawki replied:
nvm misreading, saw "version.conf" and looked for it, not "db_ver" which is the databse tool thing

EpicTaru committed:
Update Server-Setup-and-Maintenance-Linux.md

So one `/` said to the other `/`, "You see that `\` over there, he don't belong in this here slash of the woods!!
